### PR TITLE
feat(projector): add flex toggling

### PR DESCRIPTION
**Pull request by punctuations**
Added:
- new command set_flex_schedule 
    - This command will change the current schedule that runs on flex days
- Added new dependency to requirements

Fixes:
- Fixed flex_schedule_off and flex_schedule_on
    - Change the path to toggle scripts

### CODE REVIEW

File: scripts/Projector/flex_schedule_off.py [hunk 1]
```diff
@@ -5,6 +5,6 @@
 
 def flex_schedule_off():
     ssh_connection = ssh.SSH(hostname, pi.username, pi.password)
-    ssh_connection.send_cmd("rm /tmp/flex")
+    ssh_connection.send_cmd("rm /home/pi/clock/public/flex-toggle")
     print("Done.")
```
Comment (tylerecouture) [inline scripts/Projector/flex_schedule_off.py:9]: Can you provide better feedback, like "Flex schedule is now OFF"?  Also does send_cmd return anything, like  success or fail? I don't remember, if it does, you should check that first.

File: scripts/Projector/flex_schedule_on.py [hunk 2]
```diff
@@ -5,6 +5,6 @@
 
 def flex_schedule_on():
     ssh_connection = ssh.SSH(hostname, pi.username, pi.password)
-    ssh_connection.send_cmd("touch /tmp/flex")
+    ssh_connection.send_cmd("touch /home/pi/clock/public/flex-toggle")
     print("Done.")
```
Comment (tylerecouture) [inline scripts/Projector/flex_schedule_on.py:9]: Ditto above

**Review by tylerecouture — CHANGES_REQUESTED** [review]
Excited to try this!  A few tweaks requested.

File: scripts/Projector/flex_schedule_on.py (hunk 2 above)
Comment (punctuations) [reply scripts/Projector/flex_schedule_on.py]: resolved in [42bd97a](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/50/commits/42bd97abdfbdc7122db7036a34528b17eded5852)

File: scripts/Projector/flex_schedule_off.py (hunk 1 above)
Comment (punctuations) [reply scripts/Projector/flex_schedule_off.py]: resolved as per [42bd97a](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/50/commits/42bd97abdfbdc7122db7036a34528b17eded5852)

**Review by tylerecouture — APPROVED** [review]

**Pull request merged**